resources/functionMap.php

```diff
@@ -5654,7 +5654,7 @@
 'is_writable' => ['bool', 'filename'=>'string'],
 'is_writeable' => ['bool', 'filename'=>'string'],
 'Iterator::current' => ['mixed'],
-'Iterator::key' => ['mixed'],
+'Iterator::key' => ['int|string|null'],
```

Comment:
Are you sure?

Reply:
iterators are allowed to return other types too (of course, to be able to use `iterator_to_array` preserving keys, you need to use `int|string` without duplicate keys, but that's because of the array, not because of the iterator)